src/main/java/com/kryptnostic/conductor/rpc/ConductorElasticsearchApi.java

```diff
@@ -39,6 +47,8 @@
 
 	Boolean initializeEntitySetDataModelIndex();
```

Comment:
Prefer primitives to boxed types. Is there a reason all the boolean variables are boxed on this interface?
https://books.google.com/books?id=ka2VUBqHiWkC&pg=PA221&lpg=PA221&dq=prefer+primitive+types+to+boxed+primitives&source=bl&ots=yZKlOeo_NZ&sig=yNFIINhohAN2D398nEI1ZQt9Zio&hl=en&sa=X&ved=0ahUKEwiX6sbZ5IHSAhWCslQKHbYoAe0Q6AEINjAE#v=onepage&q=prefer%20primitive%20types%20to%20boxed%20primitives&f=false